charts/konk/scripts/provision.sh

```diff
@@ -22,3 +22,8 @@ then
   kubectl -n {{ .Release.Namespace }} create secret generic {{ include "konk.fullname" . }}-kubeconfig \
     --from-file=/etc/kubernetes/admin.conf
 fi
+
+echo
+date
+echo done.
+sleep 300
```

Comment:
This only works for the first 300 seconds, after which a `helm upgrade -i --wait` will hang forever due to the Completed pod. I'll experiment with other solutions.